.kitchen.yml

```diff
@@ -11,182 +11,319 @@ platforms:
     run_list: selinux::disabled
   - name: centos-6.7
     run_list: selinux::disabled
-  - name: centos-7.2
+  - name: centos-7.1
     run_list: selinux::disabled
-  - name: debian-7.10
+  - name: debian-7.8
     run_list: apt::default
-  - name: debian-8.4
+  - name: debian-8.1
     run_list: apt::default
+  - name: fedora-21
   - name: fedora-22
-    run_list: yum::dnf_yum_compat
-  - name: fedora-23
-    run_list: yum::dnf_yum_compat
   - name: ubuntu-12.04
     run_list: apt::default
   - name: ubuntu-14.04
     run_list: apt::default
-  - name: ubuntu-16.04
-    run_list: apt::default
-  - name: opensuse-13.2
-  - name: omnios-r151014
-    driver:
-      box: omnios/r151014
-
+      
 suites:
   #
+  # client
+  #
+  - name: installation_client_package-50
+    run_list:
+    - recipe[mysql_test::installation_client]
+    attributes:
+      mysql:
+       version: '5.0'
+    includes: [
+      'centos-5',
```

Comment:
the includes and excludes here don't look right. They look like docker image names not the bento names